.gitignore

```diff
@@ -0,0 +1,17 @@
+# OS X temporary files that should never be committed
+
+.DS_Store
+.Trashes
+*.swp
+
+# fastlane specific
+**/fastlane/report.xml
+
+# deliver temporary files
+**/fastlane/Preview.html
+
+# snapshot generated screenshots
+**/fastlane/screenshots
+
+# scan temporary files
+**/fastlane/test_output
```

Comment:
My version of gitignore for fastlane section

```
# fastlane
fastlane/report.xml
fastlane/Preview.html
fastlane/screenshots/**/*.png
fastlane/test_output
fastlane/.env
fastlane/.env.*
./archieve
```

I don't remember why i added those a bit differently, but it is worth to check.

Reply:
Yes we don't use env variable in project folder and we are clearing the artifacts right after deployment